src/components/inputs/localized-multiline-text-input/localized-multiline-text-input.spec.js

```diff
@@ -1,283 +1,272 @@
-    });
-    it('should open all languages', () => {
-      expect(wrapper).toRender({ name: 'foo.en' });
-      expect(wrapper).toRender({ name: 'foo.de' });
-      expect(wrapper).toRender({ name: 'foo.es' });
-    });
-    it('should not render the collapse button', () => {
-      expect(wrapper.find(TranslationInput).last()).toHaveProp(
-        'languagesControl',
-        null
-      );
-    });
+it('should call onFocus when the input is focused', () => {
+  const onFocus = jest.fn();
+  const { getByLabelText } = renderLocalizedMultilineTextInput({ onFocus });
+  const input = getByLabelText('EN');
+  input.focus();
+  expect(input).toHaveFocus();
+  expect(onFocus).toHaveBeenCalled();
+});
+
+it('should call onBlur when input loses focus', () => {
+  const onBlur = jest.fn();
+  const { getByLabelText } = renderLocalizedMultilineTextInput({ onBlur });
+  const input = getByLabelText('EN');
+  input.focus();
+  expect(input).toHaveFocus();
+  input.blur();
+  expect(input).not.toHaveFocus();
+  expect(onBlur).toHaveBeenCalled();
+});
+
+describe('when input is collapsed', () => {
+  it('should allow changing value of the input', () => {
+    const { getByLabelText } = renderLocalizedMultilineTextInput();
+    const event = { target: { value: 'I want chicken' } };
+    const input = getByLabelText('EN');
+    fireEvent.focus(input);
+    fireEvent.change(input, event);
+    fireEvent.keyDown(input, { key: 'Enter' });
+    fireEvent.keyUp(input, { key: 'Enter' });
+    expect(input.value).toEqual('I want chicken');
   });
+});
 
-  describe('when autofocus is activated', () => {
-    beforeEach(() => {
-      props = createTestProps({
-        name: 'foo',
-        value: { en: '', de: '', es: '' },
-        isAutofocussed: true,
-        areLanguagesDefaultOpened: true,
-      });
-      wrapper = shallow(<LocalizedMultilineTextInput {...props} />);
-    });
-    it('should focus the selected languages', () => {
-      expect(wrapper.find({ name: 'foo.en' })).toHaveProp(
-        'isAutofocussed',
-        true
-      );
+describe('when input is expanded', () => {
+  it('should expand and show all language inputs when `Show all languages` is clicked', () => {
+    const { getByLabelText, getByText } = renderLocalizedMultilineTextInput();
+    getByText(/show all languages/i).click();
+    expect(getByLabelText('FR')).toBeInTheDocument();
+  });
+  it('should allow changing the french input', () => {
+    const { getByLabelText, getByText } = renderLocalizedMultilineTextInput();
+    getByText(/show all languages/i).click();
+    const event = { target: { value: 'Je veux manger du poulet' } };
+    const frenchInput = getByLabelText('FR');
+    fireEvent.focus(frenchInput);
+    fireEvent.change(frenchInput, event);
+    fireEvent.keyDown(frenchInput, { key: 'Enter' });
+    fireEvent.keyDown(frenchInput, { key: 'Enter' });
+    expect(frenchInput.value).toEqual('Je veux manger du poulet');
```

Comment:
Was this one of the cases where

`expect(frenchInput).toHaveAttribute('value', 'Je veux manger du poulet');`

did not work?

Reply:
yeah because `textarea` doesn't have a value in the dom 🤔

Reply:
Cool, all good. Just wanted to check again 👍